substratevm/src/com.oracle.svm.driver/src/com/oracle/svm/driver/BundleSupport.java

```diff
@@ -108,7 +124,14 @@ final class BundleSupport {
     private final BundleProperties bundleProperties;
 
     static final String BUNDLE_OPTION = "--bundle";
-    static final String BUNDLE_FILE_EXTENSION = ".nib";
+    static final String BUNDLE_FILE_EXTENSION = BundleLauncher.BUNDLE_FILE_EXTENSION;
+
+    public ContainerSupport containerSupport;
+
+    static final Path CONTAINER_GRAAL_VM_HOME = Path.of("/graalvm");
+    private static final String DEFAULT_DOCKERFILE = NativeImage.getResource("/container-default/Dockerfile");
+    private static final String DEFAULT_DOCKERFILE_MUSLIB = NativeImage.getResource("/container-default/Dockerfile_muslib_extension");
```

Comment:
Please define a static private method in `BundleSupport`
```java
private static String getDockerfile(String name) {
    return NativeImage.getResource("/container-default/" + name);
}
```
and use if for `DEFAULT_DOCKERFILE` and `DEFAULT_DOCKERFILE_MUSLIB`.

Reply:
Addressed in commit https://github.com/oracle/graal/pull/6496/commits/e22646ba54ab57f8bd11e56b1a805c7ef3222bce